tests/tests.js

```diff
@@ -637,19 +675,26 @@ QUnit.test('processTree: head tag', function(assert) {
 QUnit.test('minimizeStyles: root html tag', function(assert) {
   var message = {
     'html': [
-        '<html',
+        '<html id="myId" ',
         'style="animation-delay: 0s; width: 5px;" ',
         '></html>'
     ],
     'frameHoles': null,
     'idToStyleIndex': {},
+    'idToStyleMap': {
+      'myId': {
+        'animation-delay': '0s',
+        'width': '5px'
+      }
+    },
     'windowHeight': 5,
     'windowWidth': 5,
+    'rootId': 'myId',
     'rootStyleIndex': 1,
     'frameIndex': '0'
   };
   minimizeStyles(message);
-  assert.equal(message.html[1], 'style="width: 5px;" ');
+  assert.equal(message.html[1], 'style="width:5px;" ');
```

Comment:
I've been playing around with it a little, and I think that even though the processing was incorrect, it didn't actually messed up any cases.

The issue was that for a the following declaration:
background-color: url(&quot;example.com&quot;);

The regexp would pick out:
name=background-color
value=url(&quot

Instead of:
name=background-color
value=url(&quot;example.com&quot;)

As a result, it would notice that the styled value differed from the unstyled value, so it wouldn't minimize anything.  The only case in which this would introduce a bug is if a property name value pair that contained a url _should_ be removed in the minimization step.  However, I think it will always be the case that an unstyled element will never have a property value that contains a url, so any property values with a url should never be removed anyway.